src/auricle/ui.cljs

```diff
@@ -70,13 +70,37 @@
               :render-row #(r/as-element [speaker-item %])
               :enableEmptySections true}])
 
-(defn export-button []
-  [touchable-highlight {:on-press #(dispatch [:export-data])}
-   [text {:style {:padding 10 :background-color "#FFDD67" :margin-bottom 10} } "Export to Paste.ee"]])
+(def RNFS (js/require "react-native-fs"))
+(def what-platform (keyword (str (.-OS (.-Platform ReactNative)))))
+(def the-relative-file-path "auricle-saved.txt")
+(def the-file-path (str (case what-platform ;;.-DocumentDirectoryPath
+                           :android (.-ExternalDirectoryPath RNFS)
+                           :ios (.-LibraryDirectoryPath RNFS)) "/" the-relative-file-path))
+(defn rnfs-write-to-file [speakers] (.writeFile RNFS the-file-path (str speakers) "utf8"))
+(defn alert-about-write-failed [e] (.alert (.-Alert ReactNative)
```

Comment:
Alerts are typically used in the app, consider abstracting it out in a convenient function.